### PR TITLE
add dist train perf report

### PR DESCRIPTION
**Pull request by typhoonzero**

### CODE REVIEW

File: fluid/PaddleCV/image_classification/dist_train/README.md [hunk 1]
```diff
@@ -90,4 +92,11 @@ The default resnet50 distributed training config is based on this paper: https:/
...
 ### Performance
 
-TBD
+The below figure shows fluid distributed training performances. We did these on a 4-node V100 GPU cluster,
+each has 8 V100 GPU card, with total of 32 GPUs. All modes can reach the "state of the art (choose loss scale carefully when using fp16 mode)" of ResNet50 model with imagenet dataset. The Y axis in the figure shows
+the images/s while the X-axis shows the number of GPUs.
+
+<p align="center">
+<img src="../images/imagenet_dist_performance.png" width=528> <br />
+Performance of Fluid Distributed Training
```
Comment (guru4elephant) [inline fluid/PaddleCV/image_classification/dist_train/README.md]: this title does not match the content of the image, maybe change to 'Performance of Multiple-GPU Training of Resnet50 on Imagenet'?

File: fluid/PaddleCV/image_classification/dist_train/README.md [hunk 2]
```diff
@@ -90,4 +92,11 @@ The default resnet50 distributed training config is based on this paper: https:/
 
 ### Performance
 
-TBD
+The below figure shows fluid distributed training performances. We did these on a 4-node V100 GPU cluster,
+each has 8 V100 GPU card, with total of 32 GPUs. All modes can reach the "state of the art (choose loss scale carefully when using fp16 mode)" of ResNet50 model with imagenet dataset. The Y axis in the figure shows
+the images/s while the X-axis shows the number of GPUs.
+
+<p align="center">
+<img src="../images/imagenet_dist_performance.png" width=528> <br />
```
Comment (guru4elephant) [inline fluid/PaddleCV/image_classification/dist_train/README.md:100]: Speedup ratio should be specified in  a different table.

Comment (typhoonzero) [reply fluid/PaddleCV/image_classification/dist_train/README.md:100]: Done.

File: fluid/PaddleCV/image_classification/dist_train/README.md (hunk 1 above)
Comment (typhoonzero) [reply fluid/PaddleCV/image_classification/dist_train/README.md]: Done.

**Review by guru4elephant — APPROVED** [review]
LGTM. should add more multi-gpu training benchmark for other models.

**Pull request merged**